test/acceptance/specs/ProblemReportUISpec.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.specs
+
+import acceptance.pages.{ProblemReportPage, ThankYouPage}
+import acceptance.specs.BaseSpec
+import acceptance.specs.tags.{UiTests}
+
+class ProblemReportUISpec extends BaseSpec {
```

Comment:
Minor point but these pages are now referred to in code as "ReportProblem", e.g. `ReportProblemPage`, `ReportProblemController`. The standalone acceptance tests might be out of date on naming, I'd recommend renaming here.